src/client.ts

```diff
@@ -162,6 +162,7 @@ export class VocdoniSDKClient {
   public url: string;
   public wallet: Wallet | Signer | null;
   public electionId: string | null;
+  public explorerUrl: string;
```

Comment:
Maybe

`public readonly explorerUrl: string;`

Would be better, but this should also be done for other already existing properties of the class